packages/lib-react-native/ios/TonClientModule.mm

```diff
@@ -0,0 +1,111 @@
+
+#import "TonClientModule.h"
+#import "tonclient.h"
+
+@implementation TonClientModule
+{
+  bool hasListeners;
+}
+-(void)startObserving {
+    hasListeners = YES;
+}
+
+-(void)stopObserving {
+    hasListeners = NO;
+}
+
+
+- (dispatch_queue_t)methodQueue
+{
+    return dispatch_queue_create("react-native.client.ton", DISPATCH_QUEUE_SERIAL);
```

Comment:
Probably you wanted to reimplement the file??

Reply:
Indeed! Fixed.